src/framework/preferences/js/AuxBuilder.js

```diff
@@ -576,25 +587,13 @@ https://github.com/fluid-project/infusion/raw/main/Infusion-LICENSE.txt
     };
 
     fluid.defaults("fluid.prefs.auxBuilder", {
-        gradeNames: ["fluid.prefs.auxSchema", "{that}.buildAuxiliary"],
+        gradeNames: ["fluid.component"],
         mergePolicy: {
             elementCommonOptions: "noexpand"
         },
         // A list of all requested preferences, to be supplied by an integrator or concrete grade.
         // Typically provided through the `fluid.prefs.builder` grade.
         // requestedPrefs: [],
-        invokers: {
-            // An invoker used to generate a set of grades that comprise a
-            // final version of the auxiliary schema to be used by the PrefsEditor
-            // builder.
-            buildAuxiliary: {
-                funcName: "fluid.prefs.auxBuilder.buildAuxiliary",
-                args: [
-                    "{that}.options.auxiliarySchemaIndex",
-                    "{that}.options.requestedPrefs"
-                ]
-            }
```

Comment:
It's great that you could remove this and `auxiliarySchemaIndex` from this grade. Seems a lot cleaner like this.